doc/user_guide/big_data.rst

```diff
@@ -433,6 +433,35 @@ compute the result of all functions that are affected by the axes
 parameters. This is the reason why e.g. the result of
 :py:meth:`~._signals.signal1d.Signal1D.shift1D` is not lazy.
 
+.. _remote_cluster-label:
+
+Running on Remote Cluster
+-------------------------
+
+Running computation on remote cluster can be done easily using ``dask_jobqueue``
+
+.. code-block:: python
```

Comment:
Move this example here, the example of the gallery needs to be run and I am not that it would be possible/easy on CI.

Reply:
Yea I think maybe the distributed workflow would run but not the slurm workflow so you are right we should move it.